packages/cli/src/commands/pull.ts

```diff
@@ -31,6 +38,7 @@ const help = () => {
   )}    Path to the global ${'`.vercel`'} directory
     -d, --debug                    Debug mode [off]
     --env [filename]               The file to write Development Environment Variables to [.env]
+    --target ${getEnvTargetPlaceholder()}
```

Comment:
I'm not sure about `--target`. Should this be `--environment` to match the `vc env add [name] [environment] [gitbranch]` name?

Reply:
I think that `--environment` will be confusing next to `--env`. I went with `--target` here because that what `deploy` used, but I see now that's deprecated. I'm definitely open to other naming ideas for this.

Reply:
Maybe we can deprecate `--env` in favor of `--env-file` to make it more distinct from `--environment`?

Reply:
I think I like the second option better, otherwise we would need to rewrite all the docs and `vc env --help`.

Notice that all the examples use "Environment" 

<img width="765" alt="image" src="https://user-images.githubusercontent.com/229881/160473663-96b9208d-2e35-4bc3-b7d1-4cf6559467f6.png">


https://vercel.com/docs/cli#commands/env

Reply:
That makes sense. I updated this PR to deprecate `--env` for `--env-file` and renamed `--target` to `--environment`.